examples/GrpcService/Startup.cs

```diff
@@ -39,17 +39,10 @@ public void ConfigureServices(IServiceCollection services)
                     .ConfigureResource(r => r.AddService(this.Configuration.GetValue("ServiceName", defaultValue: "otel-test")!))
                     .AddAspNetCoreInstrumentation();
 
-                // Switch between Jaeger/Zipkin/Console by setting UseExporter in appsettings.json.
+                // Switch between Zipkin/Console by setting UseExporter in appsettings.json.
```

Comment:
It would be good to replace Jaeger with OTLP Exporter here.

Reply:
Never mind. This already has an example of OTLP Epxorter.

Reply:
Not needed to be done in this PR.

Reply:
Updated: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4788/commits/7cc160542c96f9edf6689264d28dfd44667161c9.